Formula/snap.rb

```diff
@@ -0,0 +1,45 @@
+class Snap < Formula
+  desc "Tool to work with .snap files"
+  homepage "https://snapcraft.io/"
+  url "https://github.com/snapcore/snapd/releases/download/2.46/snapd_2.46.vendor.tar.xz"
+  sha256 "c4f532018ca9d2a5f87a95909b3674f8e299e97ba5cb5575895bcdd29be23db3"
+  license "GPL-3.0-only"
+  head "https://github.com/snapcore/snapd.git"
+
+  depends_on "go" => :build
+  depends_on "squashfs"
+
+  def install
+    ENV["GOPATH"] = buildpath
```

Comment:
Can this work without setting GOPATH?

Reply:
It cannot. See @anonymouse64 answer below.